.github/workflows/api-json.yml

```diff
@@ -21,8 +21,10 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
-      - name: Installing directory tree dependency
-        run: npm i -g directory-tree
+      - name: Installing yarn
+        run: npm install --global yarn
```

Comment:
can you try skipping yarn - if exists in path then delete this line

Reply:
Done 👍